examples/kubernetes/tcp-routing.yaml

```diff
@@ -146,3 +146,20 @@ spec:
   - backendRefs:
     - name: bar
       port: 3002
+---
```

Comment:
Nit: prefer not adding SP to this example, as the other routing examples don't contains SP, and the SP will result TCPRoute to fail.